data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/S3Source.java

```diff
@@ -99,6 +103,12 @@ public void start(Buffer<Record<Event>> buffer) {
         sqsService = new SqsService(acknowledgementSetManager, s3SourceConfig, s3Service, pluginMetrics);
```

Comment:
Should we disable the SQS service? I think in some cases we want to scan only.

Reply:
Thanks David for your review, we have made sqs configuration as an optional in pipeline.